### PR TITLE
Ec2 instance types

### LINKED ISSUE #331: Update EC2 machine specs
The following info is out of date:

https://github.com/Kitware/HPCCloud/blob/master/src/panels/run/machines.js

### PR DESCRIPTION
**Pull request by cjh1**
Fixes #331

### CODE REVIEW

**Comment by cjh1** [comment]
@TristanWright PTAL

File: src/panels/run/RunEC2.js [hunk 1]
```diff
@@ -66,12 +83,21 @@ export default React.createClass({
...
+        { family }
+      </option>;
     var machineMapper = (machine, index) =>
       <option
         key={machine.id}
         value={index}
       >
-        { `${machine.name} - ${machine.cpu} core${machine.cpu > 1 ? 's' : ''} - ${machine.memory}GB ${machine.gpu ? ' + GPU' : ''} - ${machine.storage}` }
+        { `${machine.id} - ${machine.cpu} core${machine.cpu > 1 ? 's' : ''} - ` +
+          `${machine.memory}GB ${machine.gpu ? ' + GPU' : ''} - ${machine.storage}` +
```
Comment (TristanWright) [inline src/panels/run/RunEC2.js]: memory reads as `15GiBGB` is the second GB redundant?

**Comment by TristanWright** [comment]
rebased to test, LGTM 👍

File: src/panels/run/RunEC2.js (hunk 1 above)
Comment (cjh1) [inline src/panels/run/RunEC2.js]: Yes, I push a fix

**Pull request merged**